vm-linux-helpers.cmake

```diff
@@ -227,15 +227,26 @@ endfunction(UpdateDtbFromInitrd)
 # by the vm project
 # dest_file: caller variable which is set with the kernel location
 function(GetDefaultLinuxKernelFile dest_file)
-    set(${dest_file} ${VM_LINUX_PROJECT_DIR}/images/kernel/default_bzimage_4.8.16 PARENT_SCOPE)
+  if(KernelSel4ArchX86_64)
+    set(${dest_file} ${VM_LINUX_PROJECT_DIR}/images/kernel/64/default_bzimage_4.8.16 PARENT_SCOPE)
+  else()
+    set(${dest_file} ${VM_LINUX_PROJECT_DIR}/images/kernel/32/default_bzimage_4.8.16 PARENT_SCOPE)
+  endif()
 endfunction(GetDefaultLinuxKernelFile)
 
 # Function for getting the default location of the Linux guest rootfs provided
 # by the vm project
 # dest_file: caller variable which is set with the rootfs location
 function(GetDefaultLinuxRootfsFile dest_file)
+  if(KernelSel4ArchX86_64)
     set(
-        ${dest_file} ${VM_LINUX_PROJECT_DIR}/images/rootfs/default_buildroot_rootfs.cpio
+        ${dest_file} ${VM_LINUX_PROJECT_DIR}/images/rootfs/64/default_buildroot_rootfs-bare.cpio
```

Comment:
can you use something like `arch_size_dir` here to remove the conditional?

Reply:
Done.